tests/AbstractLexerTest.php

```diff
@@ -296,7 +288,6 @@ public function testMarkerAnnotationLocaleTr(): void
         self::assertNull($mutableLexer->lookahead);
         self::assertTrue($mutableLexer->moveNext());
         self::assertNull($mutableLexer->token);
-        self::assertNotNull($mutableLexer->lookahead);
```

Comment:
Yes, this assertion is redundant according to the annotated type assertion. But this is a test that makes sure that this very annotation is not a lie. I think we should keep the assertion despite Psalm telling us to remove it.

Reply:
I was going to reply that Psalm checks that the assertion is not buggy, but I tried `return true` unconditionally from `moveNext()`, and there was no issue :grimacing: